### PR TITLE
2.0 website

### PR DESCRIPTION
**Pull request by angeldevildev**
### Summary

2.0v of the website, i've created the header and the navbar is the same as always.

### Checks

- [x] No errors
- [x] No Bugs
- [x] Ran the command `npm run build`
- [x] Ran the command `npm run lint`
- [x] Ran the command `npm run dev`
- [x] Ran the command `npx prettier . --write`

### CODE REVIEW

File: src/app/style.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  section: `flex md:flex-row flex-col ${styles.paddingY}`,
+  sectionReverse: `flex md:flex-row flex-col-reverse ${styles.paddingY}`,
+
+  sectionImgReverse: `flex-1 flex ${styles.flexCenter} md:mr-10 mr-0 md:mt-0 mt-10 relative`,
+  sectionImg: `flex-1 flex ${styles.flexCenter} md:ml-10 ml-0 md:mt-0 mt-10 relative`,
+
+  sectionInfo: `flex-1 ${styles.flexStart} flex-col`,
+};
+
+export default styles;
```
Comment (theonewhoknocks878) [inline src/app/style.js]: For what?

File: src/ui-components/Hero/Hero.tsx [hunk 2]
```diff
@@ -1,48 +1,44 @@
...
+        Supercharge your development with a powerful library of UI components designed to improve productivity and code quality for React developers.
+      </p>
+      <div className="flex flex-row items-center gap-5">
+        <div className="bg-white text-black rounded-md">
+          <Link
+            href="/"
+            className={buttonVariants({ className: "px-6", size: "lg" })}
+          >
+            Get Stared
+          </Link>
```
Comment (theonewhoknocks878) [inline src/ui-components/Hero/Hero.tsx:29]: I like the new changes

File: src/ui/globe.tsx
Comment (theonewhoknocks878) [inline src/ui/globe.tsx:1]: Are we gonna use this?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ _Add more context to describe the changes..._
 
 [Example](www.google.com)
 
-### Checks
+### Required Checks
```
Comment (theonewhoknocks878) [inline .github/pull_request_template.md:13]: Good idea

**Review by theonewhoknocks878 — CHANGES_REQUESTED** [review]
Remove the style.js

File: src/app/style.js (hunk 1 above)
Comment (angeldevildev) [reply src/app/style.js]: Deleted.

File: src/ui/globe.tsx
Comment (angeldevildev) [reply src/ui/globe.tsx:1]: Probably

File: .github/pull_request_template.md (hunk 3 above)
Comment (angeldevildev) [reply .github/pull_request_template.md:13]: Or people don't do all the checks that are required

**Review by theonewhoknocks878 — COMMENTED** [review]
Ok

**Pull request merged**